addons/web/static/tests/fields/basic_fields_tests.js

```diff
@@ -479,6 +479,87 @@ QUnit.module('basic_fields', {
         form.destroy();
     });
 
+    QUnit.module('FieldNumeric');
+
+    QUnit.test('numeric field: fields with keydown on numpad decimal key', async function (assert) {
+        assert.expect(6);
+
+        this.data.partner.fields.float_factor_field = { string: "Float Factor", type: 'float_factor' };
```

Comment:
Sicne the new code is on NumericField, you probably only have to formally test only one of them. Since you already wrote them, this is fine though....